test/runnable/sctor.d

```diff
@@ -360,7 +360,7 @@ void test14450()
 
     { auto i = immutable S14450a(1);    assert(i.x == 2); }
     { auto i = immutable S14450b(1);    assert(i.x == 2); }
-    { auto i = immutable S14450c!()(1); assert(i.x == 2); }
+    { auto i = immutable S14450c!()(1); assert(i.x == 1); }
```

Comment:
The `==1` cases above are making a `S14450c!()`, this case is making an `immutable S14450c!()` - so they're not the same.  Why wouldn't this use the immutable ctor?